modules/frontend/src/main/scala/smithy4s_codegen/components/pages/Home.scala

```diff
@@ -22,6 +18,8 @@ object Home {
       editor.codeContent.signal
         .composeChanges(_.debounce(2000))
         .flatMap { value =>
+          PermalinkCodec.write(value)
```

Comment:
thought: this one I hate the most. Ideally I could do sth like `composeChanges(...).observe(PermalinkCodec.write: SInk)` but that ain't it.